### DISCUSSION TITLE
Regarding the new version of tensorflowv2.16.1

### DISCUSSION THREAD

**Discussion by marvie910** [ika-rwth-aachen/libtensorflow_cc#11] [Q&A]
Hello!
How do I use the repository to get a pre-built  library for the latest tensorflow version 2.16.1?

**Comment by lreiher** [comment]
Unfortunately we have not yet succeeded to support the newer TensorFlow versions. You can try to follow the [README instructions on building yourself](https://github.com/ika-rwth-aachen/libtensorflow_cc?tab=readme-ov-file#build). Try searching for `2.13.0` in the repository and add `2.16.1` where it seems to make sense to start pulling and building with the `2.16.1` source code.

The first problem you will likely run into is that the TensorFlow repository no longer provides Dockerfiles for building TensorFlow from source. Previously, these were pulled as the first step with [this script](https://github.com/ika-rwth-aachen/libtensorflow_cc/blob/main/scripts/0-download-official-dockerfiles.sh). You might be able to set up a custom development Docker image though that you can then use for the remaining steps.

Feel free to post a pull request if you have succeeded to build one of the newer versions.

Also feel free to ask for more advice if you don't really know how to get started.
